.coveragerc

```diff
@@ -757,9 +756,10 @@ omit =
     homeassistant/components/sensor/mvglive.py
     homeassistant/components/sensor/nederlandse_spoorwegen.py
     homeassistant/components/sensor/netatmo_public.py
-    homeassistant/components/sensor/netdata.py
     homeassistant/components/sensor/netdata_public.py
+    homeassistant/components/sensor/netdata.py
     homeassistant/components/sensor/neurio_energy.py
+    homeassistant/components/sensor/nmbs.py
```

Comment:
Not available.

Reply:
but my file is at homeassistant/components/sensor/nmbs.py in the diff below?